### ISSUE TITLE
Choosing Mothers Face/Tone won't change apperance

### ISSUE THREAD

**Issue by CaleighRhodes** [MAFINS/MenyooSP#665]
Hey all together,

since a month or so I have the problem that if I try to desing a new Charakter with Menyoo in Object Spooner -> Ped -> Multiplayer Models -> Freemode Female it won't apply the Facial Features of mothers. It only changes fathers faces. Its just nothing happening when I choose one of the mothers features.

Does anybody had a similar issue and could help me solve?
I tried reinstalling GTA and Menyoo several times. Tried old Menyoo, tried new Menyoo, nothing. ;(

**Comment by mistahwolf** [comment]
@CaleighRhodes, yeah this is a long-time issue with Menyoo. In the past, if you logged in on-line it would work for a while. Only recently with 'Ghost's' last update have I seen it working consistently. His or 'itsJustCurtis' version of Menyoo may fix the issue.

**Comment by SkyK0T** [comment]
> Try this: https://www.mediafire.com/file/o4c0nt2yabg2eep/Menyoo_28-01-2024.zip/file
you site http://www.gtamods4you.com/ not working
☺😊

**Comment by mistahwolf** [comment]
@The-UnXpLaiNeD yeah that version 'broke' the 'mother's face, tone' and the separate 'Tone' adjustments. The previous update fixed them. However, this version stopped aircraft from blowing up like the blimps do when selected in spooner. Still got issues with the teleport markers not activating while on foot, but working when in a car or on a motorcycle. It seems like a 'memory' thing as it 'works' initially early in a game session. But if you go somewhere else then come back, the problem kicks in. Haven't tried to see if it will load maps with more than 600 props yet to see if that works again. It's a PIA having to restart the game when it crashes if it doesn't. Anyway, thanks for sticking with this!
